eo-maven-plugin/src/main/java/org/eolang/maven/FtCached.java

```diff
@@ -73,14 +74,17 @@ public final class FtCached implements Footprint {
      * @param cache Cache root
      */
     public FtCached(final String[] labels, final Path main, final Path cache) {
-        this.code = String.join("/", labels);
+        this.code = labels.clone();
```

Comment:
`Clone` is suggested to avoid. It's better use `Arrays.copyOf()` for make a copy.

Reply:
@mximp done. BTW, are there any reasons? Is't `clone` safe enough?

Reply:
Thanks